tests/addon-template/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "addon-template",
+  "version": "0.0.0",
+  "description": "The default blueprint for ember-cli addons.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "ember test",
+    "test:all": "ember try:each"
+  },
+  "dependencies": {
+    "ember-cli-babel": "^7.7.3"
+  },
+  "devDependencies": {
+    "@ember/optional-features": "^0.7.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-cli": "~3.10.1",
```

Comment:
Yikes, this seems quite old (and actually is unsupported). Why generate with ember-cli@3.10? Shouldn't we at least use one of our supported LTS versions?

Reply:
I'd suggest using `npx ember-cli@3.24 addon addon-template --skip-npm --no-welcome` to regenerate.

Reply:
Oops I ran `ember addon addon-template` inside of this directory and seems like the global ember choose an older version. I have updated this to use a new version

Reply:
🎉